sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/FileIndexSuite.scala

```diff
@@ -416,6 +416,37 @@ class FileIndexSuite extends SharedSparkSession {
     }
   }
 
+  test("SPARK-29189: Add an option to ignore block locations when listing file") {
```

Comment:
Thank you for adding this. Since this is a new feature, please remove the prefix, `SPARK-29189: `.

Reply:
Resolved.